label_check/entrypoint.sh

```diff
@@ -57,31 +61,49 @@ main(){
         label_to_check=$label_name
 
         if [ "$label_name" = "deploy" ]; then
```

Comment:
I think that is better to use the same variable you have created before `production_label` and the same for `staging_label`

Reply:
solved